src/adaptors/zipswap/index.js

```diff
@@ -0,0 +1,64 @@
+const { request, gql } = require('graphql-request');
+const utils = require('../utils');
+
+const subgraphURL = "https://api.thegraph.com/subgraphs/name/nonamefits/zipswap";
+
+const getCurrentDate= gql`
+{
+uniswapDayDatas(orderBy:date
+  orderDirection:desc, first:1){
+    date
+  }
+}
+`
+const getDate = async () => {
+  let data = await request(subgraphURL ,getCurrentDate);
+  let date = data.uniswapDayDatas[0].date;
+  return date+"";
+}
+
+var query = gql`
+{
+  pairDayDatas(first: 5,
```

Comment:
why only first 5? i'd increase that value cause u missing some pools, like 'WETH-ZIP'.

Reply:
Initially thought 5 would get the most relevant pools, but it the volume varies too much on different days. Expanded to 10 now to cover all relevant pools.